From fb90e42c3792f4d94c75974e711be61222232235 Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Mon, 25 Dec 2023 12:21:08 +0200
Subject: [PATCH] tests: P2P: Also wait in go_neg_pbc()

Similar to other changes like this, wait a bit before
connectivity checks in go_neg_pbc().

Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
 tests/hwsim/p2p_utils.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/hwsim/p2p_utils.py b/tests/hwsim/p2p_utils.py
index 31d0f74a4..fe115e0ab 100644
--- a/tests/hwsim/p2p_utils.py
+++ b/tests/hwsim/p2p_utils.py
@@ -374,6 +374,7 @@ def go_neg_pbc(i_dev, r_dev, i_intent=None, r_intent=None, i_freq=None,
         raise Exception("go_neg_init_pbc thread failed")
     logger.debug("i_res: " + str(i_res))
     logger.info("Group formed")
+    time.sleep(0.1)
     hwsim_utils.test_connectivity_p2p(r_dev, i_dev)
     i_dev.dump_monitor()
     return [i_res, r_res]
-- 
2.43.0

